src/components/Cart/Cart.tsx

```diff
@@ -119,11 +125,80 @@ const Cart = () => {
             <p className="text-text-paragraph">Total</p>
             <p className="text-text-paragraph">$1400</p>
           </div>
-          <button className="text-text-paragraph bg-logo-orange relative bottom-24 w-24 rounded-3xl">
+          <button
+            className="text-text-paragraph bg-logo-orange relative bottom-24 w-24 rounded-3xl"
+            onClick={() => setModal(!modal)}
+          >
             Pagar
           </button>
         </div>
       </div>
+      {modal ? (
+        <Modal>
+          <div className="justify-center w-[100%] flex flex-col gap-4">
```

Comment:
we should put the entire form in another component.

Reply:
done